apps/studio/src/components/tableview/TableTable.vue

```diff
@@ -927,8 +927,14 @@ export default Vue.extend({
     cellCloneRow(_e, cell) {
       const row = cell.getRow()
       const data = { ...row.getData() }
+      const dataParsed = Object.keys(data).reduce((acc, d) => {
+        if (!this.primaryKeys.find(element => element === d)) {
```

Comment:
This should just use `includes` -> `if(!this.primaryKeys?.includes(d))`

Reply:
Includes it is!